src/codeflare_sdk/cluster/auth.py

```diff
@@ -161,8 +162,23 @@ def config_check() -> str:
     """
     global config_path
     global api_client
+    home_directory = os.path.expanduser("~")
     if config_path == None and api_client == None:
-        config.load_kube_config()
+        if os.path.isfile("%s/.kube/config" % home_directory):
+            try:
+                config.load_kube_config()
+            except Exception as e:  # pragma: no cover
+                print(e)
```

Comment:
Similarly to below, if an exception occurs we probably don't want to just print it and keep going. I would replace `print(e)` here with `_kube_api_error_handling(e)` (import from `utils/kube_api_helpers`)